packages/office-ui-fabric-react/src/components/Callout/Callout.types.ts

```diff
@@ -102,7 +102,13 @@ export interface ICalloutProps extends React.HTMLAttributes<HTMLDivElement> {
   preventDismissOnLostFocus?: boolean;
 
   /**
-   *If defined, then takes priority over preventDismissOnLostFocus, preventDismissOnResize,
+   * If true then the callout will dismiss when the window gets focus
+   * @defaultvalue false
+   */
+  shouldDismissOnWindowFocus?: boolean;
```

Comment:
@joschect should we add this prop to Callout HOCs? We are using the new behavior on ContextualMenus

Reply:
It's already added! You can pass it through to the underlying callout like

```
<ContextualMenu
    calloutProps = {{shouldDismissOnWindowFocus: true }}
/>
```